docs/resources/project.md

```diff
@@ -80,6 +98,34 @@ The following arguments are supported:
 
 * `packages_enabled` - (Optional) Enable packages repository for the project.
 
+* `push_rules` (Optional) Manage push rules for your GitLab project.
```

Comment:
This is duplicated?

Reply:
Nope, this page is the resource. The earlier one was the data source.